x-pack/plugins/security_solution/public/timelines/components/formatted_ip/index.tsx

```diff
@@ -5,24 +5,35 @@
  * 2.0.
```

Comment:
I am wondering if you can give me a favor here and remove the useMemo in line 150. there is no need of it. It is just a string.

```
const key = useMemo(
    () =>
      `address-links-draggable-wrapper-${getUniqueId({
        contextId,
        eventId,
        fieldName,
        address,
      })}`,
    [address, contextId, eventId, fieldName]
  );
```
in case just be that 

```
const key = `address-links-draggable-wrapper-${getUniqueId({
        contextId,
        eventId,
        fieldName,
        address,
      })}`;
```

Reply:
Done!